lib/gollum/public/gollum/javascript/editor/gollum.editor.js

```diff
@@ -306,6 +306,11 @@
               formData = new FormData();
           formData.append('file', file);
           
+          if (Object.prototype.toString.call(file) != '[object File]') {
+            $editorBody.removeClass('uploading');
+            return;
+          }
+          
```

Comment:
Looks like this commit sneaked in from another branch.

Reply:
Thanks, just fixed this through a rebase.